web/client/utils/MapInfoUtils.js

```diff
@@ -152,12 +152,16 @@ const MapInfoUtils = {
         const center = {x: lngCorrected, y: props.point.latlng.lat};
         let centerProjected = CoordinatesUtils.reproject(center, 'EPSG:4326', props.map.projection);
         let bounds = MapInfoUtils.getProjectedBBox(centerProjected, resolution, rotation, size, null);
+        let queryLayers = layer.name;
+        if (layer.queryable) {
```

Comment:
That's the flag I was referring too. It's coming from WMS capabilities to tell you if a layer is queryable or not.